.github/workflows/license-check.yml

```
@@ -4,15 +4,77 @@ on:
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: worker/go.mod
+          check-latest: true
+          cache-dependency-path: |
+            worker/go.sum
+      - uses: reearth/actions/license-checker/go@ec26fbcf84f0e954a6ba3511859b18e3d2272c18
+        with:
+          subdirectory: worker
   web-license-check:
     name: Web License Check
+    if: needs.prepare.outputs.server == 'true' || github.event_name == 'workflow_dispatch'
```

actionlint (1.7.4)

property "prepare" is not defined in object type {}

(expression)